<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <property name="accessibleDescription">
            <string>HOLA!</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Nombre Correci√≥n</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineEdit"/>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_3">
                            <property name="styleSheet">
                                <string notr="true">color:red</string>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,1">
                    <property name="spacing">
                        <number>6</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Carpeta de archivos</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineEdit_2">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                    <horstretch>1</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background:transparent</string>
                            </property>
                            <property name="text">
                                <string>--</string>
                            </property>
                            <property name="frame">
                                <bool>false</bool>
                            </property>
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pushButton">
                            <property name="text">
                                <string>Explorar...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>10</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <widget class="QLabel" name="label_4">
                            <property name="text">
                                <string>Rubrica</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pushButton_2">
                            <property name="enabled">
                                <bool>false</bool>
                            </property>
                            <property name="text">
                                <string>Crear Nuevo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pushButton_3">
                            <property name="enabled">
                                <bool>false</bool>
                            </property>
                            <property name="text">
                                <string>Cargar</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="enabled">
                        <bool>false</bool>
                    </property>
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>Dialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>Dialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
